content/docs/2.0/scalers/azure-log-analytics.md

```diff
@@ -1,208 +1,359 @@
+
+### Example
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kedaloganalytics
+  namespace: kedaloganalytics
+  labels:
+    app: kedaloganalytics
+type: Opaque
+data:
+  tenantId: "QVpVUkVfQURfVEVOQU5UX0lE" #Base64 encoded Azure Active Directory tenant id
+  clientId: "U0VSVklDRV9QUklOQ0lQQUxfQ0xJRU5UX0lE" #Base64 encoded Application id from your Azure AD Application/service principal
+  clientSecret: "U0VSVklDRV9QUklOQ0lQQUxfUEFTU1dPUkQ=" #Base64 encoded Password from your Azure AD Application/service principal
+  workspaceId: "TE9HX0FOQUxZVElDU19XT1JLU1BBQ0VfSUQ=" #Base64 encoded Log Analytics workspace id
+---
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: trigger-auth-kedaloganalytics
+  namespace: kedaloganalytics
+spec:
+  secretTargetRef:
+    - parameter: tenantId
+      name: kedaloganalytics
+      key: tenantId
+    - parameter: clientId
+      name: kedaloganalytics
+      key: clientId
+    - parameter: clientSecret
+      name: kedaloganalytics
+      key: clientSecret
+    - parameter: workspaceId
+      name: kedaloganalytics
+      key: workspaceId
+---
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: kedaloganalytics-consumer-scaled-object
+  namespace: kedaloganalytics
+  labels:
+    deploymentName: kedaloganalytics-consumer
+spec:
+  scaleTargetRef:
+    kind: #Optional: Default: Deployment, Available Options: ReplicaSet, Deployment, DaemonSet, StatefulSet
+    name: kedaloganalytics-consumer
+  pollingInterval: 30
+  cooldownPeriod: 30
+  minReplicaCount: 1
+  maxReplicaCount: 10
+  triggers:
+  - type: azure-log-analytics
+    metadata:
+      query: |
+        let AppName = "web";
+        let ClusterName = "demo-cluster";
+        let AvgDuration = ago(10m);
+        let ThresholdCoefficient = 0.8;
+        Perf
+        | where InstanceName contains AppName
+        | where InstanceName contains ClusterName
+        | where CounterName == "cpuUsageNanoCores"
+        | where TimeGenerated > AvgDuration
+        | extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+        | summarize MetricValue=round(avg(CounterValue)) by CounterName, AppName
+        | join (Perf 
+                | where InstanceName contains AppName
+                | where InstanceName contains ClusterName
+                | where CounterName == "cpuLimitNanoCores"
+                | where TimeGenerated > AvgDuration
+                | extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+                | summarize arg_max(TimeGenerated, *) by AppName, CounterName
+                | project Limit = CounterValue, TimeGenerated, CounterPath, AppName)
+                on AppName
+        | project MetricValue, Threshold = Limit * ThresholdCoefficient
+      threshold: "1900000000"
+    authenticationRef:
+      name: trigger-auth-kedaloganalytics
+```
+
+**Managed identity based authentication:**
```

Comment:
I would move this section before the example above so that they can see all options first, before diving into the details.

Reply:
Done in last commit.